### PR TITLE
[HK] fixes NLS Warning for cancel_tooltip

### PR DESCRIPTION
**Pull request by fgdrf**
Change-Id: I7b25b4dbc3f9810b9ee461d8df26b4721a5d6c57
Signed-off-by: Frank Gasdorf <fgdrf@users.sourceforge.net>

### CODE REVIEW

**Comment by fgdrf** [comment]
@sschulz92 this fixes the warning `Warning: NLS unused message: cancel_tooltip in: org.locationtech.udig.ui.internal.message` and updates the source files with the current formatter settings

Would be great if you can have a look at ;)

File: plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java [hunk 1]
```diff
@@ -13,204 +13,359 @@
...
+    public static String cancel_tooltip;
+
     public static String orientation_horizontal_label;
+
     public static String orientation_horizontal_image;
-//    public static String orientation_horizontal_description;
-//    public static String orientation_horizontal_tooltip;
-    
+    // public static String orientation_horizontal_description;
+    // public static String orientation_horizontal_tooltip;
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java:330]: Remove those if not needed?

File: plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java [hunk 2]
```diff
@@ -13,204 +13,359 @@
...
+
     public static String orientation_vertical_label;
+
     public static String orientation_vertical_image;
-//    public static String orientation_vertical_description;
-//    public static String orientation_vertical_tooltip;
-    
+    // public static String orientation_vertical_description;
+    // public static String orientation_vertical_tooltip;
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java:336]: Remove those if not needed?

File: plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java [hunk 3]
```diff
@@ -13,204 +13,359 @@
...
+
     public static String orientation_single_label;
+
     public static String orientation_single_image;
-//    public static String orientation_single_description;
-//    public static String orientation_single_tooltip;
-    
+    // public static String orientation_single_description;
+    // public static String orientation_single_tooltip;
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java:342]: Remove those if not needed?

File: plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java [hunk 4]
```diff
@@ -13,204 +13,359 @@
...
+
     public static String orientation_automatic_label;
+
     public static String orientation_automatic_image;
-//    public static String orientation_automatic_description;
-//    public static String orientation_automatic_tooltip;
-    
+    // public static String orientation_automatic_description;
+    // public static String orientation_automatic_tooltip;
```
Comment (sschulz92) [inline plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/internal/Messages.java:348]: Remove those if not needed?

**Review by sschulz92 — CHANGES_REQUESTED** [review]

**Pull request merged**